core/templates/pages/contributor-dashboard-page/services/question-suggestion-backend-api.service.spec.ts

```diff
@@ -85,9 +84,8 @@ describe('Question Suggestion Backend Api Service', () => {
     let successHandler = jasmine.createSpy('success');
     let failHandler = jasmine.createSpy('fail');
 
-    let skillDifficulty = new SkillDifficulty('test_id', 'test_description', 1);
     qsbas.submitSuggestionAsync(
-      question as unknown as Question, associatedSkill, skillDifficulty, [{
+      question as unknown as Question, associatedSkill, 1, [{
```

Comment:
can you remove this unknown and add file name in typescript_checks.py

Reply:
done